h2o-docs/src/product/data-science/algo-params/lambda_search.rst

```diff
@@ -7,7 +7,7 @@
 Description
 ~~~~~~~~~~~
 
-Setting ``lambda_search`` to TRUE enables efficient and automatic search for the optimal value of the ``lambda`` parameter. When enabled, GLM will first fit a model with maximum regularization (highest lambda value) and then keep decreasing it at each step until it reaches the minimum lambda or until overfitting occurs. The resulting model is based on the best lambda value. 
+Setting ``lambda_search`` to TRUE enables efficient and automatic search for the optimal value of the ``lambda`` parameter. When enabled, GLM will first fit a model with maximum regularization (highest lambda value) and then keep decreasing it at each step until it reaches the minimum lambda or until overfitting occurs. (Note that GLM will automatically calculate the minimum lambda value unless a value for ``lambda`` is specified. In that case, the specified ``lambda`` value becomes the minimum lambda value.) The resulting model is based on the best lambda value. 
```

Comment:
lambda_min_ratio modifies the lambda min. 
if you supply your own lambda(s), the lambda search os only done over the provided lambdas.